src/ReactVapor.ts

```diff
@@ -64,6 +65,7 @@ export interface IReactVaporState {
     collapsibleContainers?: ICollapsibleContainerState[];
     searchBars: ISearchBarState[];
     flippables?: IFlippableState[];
+    textAreas: ITextAreaState[];

```

Comment:
shouldn't this be optional like the others?

Reply:
good catch